chartbeat.php

```diff
@@ -286,28 +305,34 @@ function add_chartbeat_head() {
 	echo "\n<script type=\"text/javascript\">var _sf_startpt=(new Date()).getTime()</script>\n";
 }
 
+function add_chartbeat_config(){
+		
+		$cb_configs = chartbeat_configs();?>
+<script type="text/javascript">
+			var _sf_async_config={};
+			_sf_async_config.uid = <?php echo $cb_configs["uid"]; ?>;
+			_sf_async_config.domain = "<?php echo $cb_configs["domain"]; ?>";
+			_sf_async_config.useCanonical = <?php echo $cb_configs["use_canonical"]; ?>;
+<?php
+		$enable_newsbeat = get_option('chartbeat_enable_newsbeat');
+		if ($enable_newsbeat) { ?>
+			_sf_async_config.authors = "<?php echo $cb_configs["author"] ?>";
+			_sf_async_config.sections = "<?php echo $cb_configs["sections"]?>";
+	<?php }
+
```

Comment:
The JS insertion is pretty specific, but I get your point about the mismatched tags. I'll remove it.